standardtracer/standardcontext.go

```diff
@@ -0,0 +1,48 @@
+package standardtracer
+
+import "sync"
+
+type StandardContext struct {
```

Comment:
This is low-importance since the whole `standardtracer` thing is just a convenience for tracing system authors, but FWIW this version of `standardtracer` is way less flexible since it hardcodes this Dapper-like `StandardContext` struct. We can push the old `TraceContext` in here and make everything way more generic if there's a demand. I didn't think it was worth the time and effort at this point in time.